spellcheck.py

```diff
@@ -2,13 +2,13 @@
 from collections import Counter
 import string
 import ast
-from flask import Flask, render_template, request
+from flask import Flask, jsonify, render_template, request
 
 
 def read_file():
     words = []
-    for i in range(1, 627):
-        with open('for_irish\irish_list_'+str(i)+'.txt', 'r', encoding='utf-8') as file:
+    for i in range(1, 15):
```

Comment:
why have you used less amount of data for English than Irish? would you think the frequency list would be good for the data given to training for English and contain all words?